vita3k/gui/src/live_area.cpp

```diff
@@ -1062,7 +1062,7 @@ void draw_live_area_screen(GuiState &gui, EmuEnvState &emuenv) {
     const auto SELECT_SIZE = ImVec2(GATE_SIZE.x - (10.f * SCALE.x), GATE_SIZE.y - (5.f * SCALE.y));
     const auto SELECT_POS = ImVec2(GATE_POS.x + (5.f * SCALE.y), GATE_POS.y + (2.f * SCALE.y));
 
-    const auto BUTTON_SIZE = ImVec2(72.f * SCALE.x, 30.f * SCALE.y);
+    const auto BUTTON_SIZE = ImVec2(80.f * SCALE.x, 55.f * SCALE.y);
```

Comment:
no need change it, problem here is not size, is how button size is scalled on android, because code for respect aspect ratio is disable on android
i not have this problem in my side
Buton size is all time same and fine

Reply:
in my phone that button was hard to press, most often vita3k notification got pressed instead esc.
or should i move a bit lower instead resize it?